app/controllers/miq_ae_class_controller.rb

```diff
@@ -873,10 +873,14 @@ def fields_form_field_changed
           page << javascript_hide("field_default_value")
           page << javascript_show("field_password_value")
           page << "$('#field_password_value').val('');"
+          session[:field_data]["default_value"] =
```

Comment:
@pkomanek we should normalize keys in session[:field_data], i see some are symbols https://github.com/pkomanek/manageiq/blob/3aad7e1a1ec1984e2ff53734286ccdef7dcc8c18/app/controllers/miq_ae_class_controller.rb#L2129-L2130 and in some places they are strings.